src/wazuh_modules/wmodules.h

```diff
@@ -17,8 +17,8 @@
 #include "config/config.h"
 #include "wmodules_def.h"
 
-#define WM_DEFAULT_DIR  BUILDDIR(HOMEDIR,"/wodles")       // Default modules directory.
-#define WM_STATE_DIR    BUILDDIR(HOMEDIR,"/var/wodles")   // Default directory for states.
+#define WM_DEFAULT_DIR  "wodles"                   // Default modules directory.
```

Comment:
This `WM_DEFAULT_DIR` is not used in the project. Remove it.

Reply:
It was removed in this commit fcff159439fdcee38c8870c942baacb153c0954f